### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by ameypv-rudder**
> Added readme

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
## Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added unit tests for the code
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,34 +10,40 @@
...
 
-\*\*Describe the key features, if necessary.\*\*
+This library lets you encrypt and decrypt PII including those that are stored in cookies. Refer to the library's code [here](https://github.com/rudderlabs/rudder-libraries/tree/main/libraries/encrypt/v1).
 
-## Getting started
+## User agent parser
 
-\*\*Describe how to use the software.\*\*
+The [Parse user agent transformation template](https://rudderstack.com/docs/features/transformations/templates/#parse-user-agent) imports this library to add the user's browser, engine, OS, device, and CPU-related information to the events. Refer to the library's code [here](https://github.com/rudderlabs/rudder-libraries/tree/main/libraries/userAgentParser/v1).
+
```
Comment (Jayachand) [inline README.md:39]: paste code here rather than navigating to docs

File: README.md [hunk 2]
```diff
@@ -10,34 +10,40 @@
...
   </b>
 </p>
 
 ---
 
-# \*\*Repo Name\*\*
+# RudderStack libraries
 
-\*\*Repo description\*\*
+RudderStack provides some libraries out of the box which you can directly import into your transformations. This repo contains the relevant code and implementation-specific details for these libraries.
```
Comment (Jayachand) [inline README.md:24]: Add import style info block
`@rs/<libname>/<version>`

Comment (ameypv-rudder) [reply README.md:24]: Done

File: README.md (hunk 1 above)
Comment (ameypv-rudder) [reply README.md:39]: Makes sense - done.

**Review by Jayachand — APPROVED** [review]

**Pull request merged**